VariableAnalysis/Lib/Helpers.php

```diff
@@ -9,6 +9,19 @@
 use PHP_CodeSniffer\Util\Tokens;
 
 class Helpers {
+  /**
+   * return int[]
+   */
+  public static function getEmptyTokens() {
+    return array_merge(
+      array_values(Tokens::$emptyTokens),
+      [
+        T_INLINE_HTML,
+        T_CLOSE_TAG,
+      ]
+    );
```

Comment:
```suggestion
$empty = Tokens::$emptyTokens;
$empty[T_INLINE_HTML] = T_INLINE_HTML;
$empty[T_CLOSE_TAG] = T_CLOSE_TAG;
return $empty;
```

Array functions like `array_merge()` and `array_values()` are "expensive".

Changing it to the above will be more efficient performance + memory-use-wise.

You could even choose to optimize it further like this:
```suggestion
static $empty;
if (isset($empty) === false) {
    $empty = Tokens::$emptyTokens;
    $empty[T_INLINE_HTML] = T_INLINE_HTML;
    $empty[T_CLOSE_TAG] = T_CLOSE_TAG;
}
return $empty;
```

It's not as if this compound array will change at any time during the run.

Reply:
Oh! Good to know. I reduced how often this function is called in #236 but I made https://github.com/sirbrillig/phpcs-variable-analysis/issues/237 to make sure I remember to clean this up.